mParticle-Singular/MPKitSingular.m

```diff
@@ -20,12 +20,16 @@ @implementation MPKitSingular
 
 // Wrapper Consts
 #define MPARTICLE_WRAPPER_NAME @"mParticle"
-#define MPARTICLE_WRAPPER_VERSION @"1.0.1"
+#define MPARTICLE_WRAPPER_VERSION @"1.1.0"
```

Comment:
why not 2?

Reply:
i can change to 2 but it's not a huge change and for internal use anyway